vllm/model_executor/layers/fused_moe/deepep_ll_prepare_finalize.py

```diff
@@ -87,11 +135,11 @@ def prepare(
             assert hidden_size % 128 == 0, \
             "DeepEP kernels quantize the inputs in blocks of shape 128"
 
-        # Quantize
-        per_act_token = a1_scale.numel() != 1 if a1_scale is not None else (
+        has_per_token_scales = a1_scale.numel(
+        ) != 1 if a1_scale is not None else (
             a2_scale.numel() != 1 if a2_scale is not None else False)
-        assert not per_act_token, (
-            "low_latency kernels don't support per-act-token quant")
+        assert not has_per_token_scales, (
+            "low_latency kernels doesn't support dispatching per-token scales")
```

Comment:
Maybe this condition should be part of `BatchedTritonOrDeepGemmExperts` logic to select which kernels to use?